src/components/ResearchGroupCardList/ResearchGroupCardList.jsx

```diff
@@ -0,0 +1,82 @@
+				courses: ["CS M146", "CS 161", "RNNs"]
+			},{
+				professor: "Adnan Darwiche", 
+				researchGroup: "Automated Reasoning Group", 
+				content: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged.",
+				courses: ["CS M146", "CS 161", "RNNs"]
+			},{
+				professor: "Adnan Darwiche", 
+				researchGroup: "Automated Reasoning Group", 
+				content: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged.",
+				courses: ["CS M146", "CS 161", "RNNs"]
+			},{
+				professor: "Adnan Darwiche", 
+				researchGroup: "Automated Reasoning Group", 
+				content: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged.",
+				courses: ["CS M146", "CS 161", "RNNs"]
+			},{
+				professor: "Adnan Darwiche", 
+				researchGroup: "Automated Reasoning Group", 
+				content: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged.",
+				courses: ["CS M146", "CS 161", "RNNs"]
+			},{
+				professor: "Adnan Darwiche", 
+				researchGroup: "Automated Reasoning Group", 
+				content: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged.",
+				courses: ["CS M146", "CS 161", "RNNs"]
+			}]
+		}
+	}
+
+  render () {
+    return (
+    	<div className="card-list">
+				{this.state.data.map(singleCard => 
+					<ResearchGroupCard
```

Comment:
Before you push things, make sure to check your console for errors.

<img width="845" alt="screen shot 2019-02-24 at 3 38 43 pm" src="https://user-images.githubusercontent.com/16346536/53307153-57021380-384a-11e9-9448-631cce0d8401.png">

You need to include a mandatory `key` prop in anything you iterate over. So you need:

```
this.state.data.map( (singleCard, i) => 
    <ResearchGroupCard
           key={i}
           courses={etc...}
           ...
       />
```

Reply:
I'm so sorry I did not check for that! I assumed there were no errors as it rendered properly. I have fixed them now!